example/main.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/aws/aws-sdk-go/aws"
+	wflambda "github.com/retgits/wavefront-lambda-go"
+)
+
+var (
+	// DefaultHTTPGetAddress Default Address
+	DefaultHTTPGetAddress = "https://checkip.amazonaws.com"
+
+	// ErrNoIP No IP found in response
+	ErrNoIP = errors.New("No IP in HTTP response")
+
+	// ErrNon200Response non 200 status code in response
+	ErrNon200Response = errors.New("Non 200 Response found")
+)
+
+var wfAgent = wflambda.NewWavefrontAgent(&wflambda.WavefrontConfig{
+	// Enabled indicates whether metrics are sent to Wavefront
+	Enabled: aws.Bool(true),
+	// Wavefront URL of the form https://<INSTANCE>.wavefront.com.
+	Server: aws.String("https://my-instance.wavefront.com"),
+	// Wavefront API token with direct data ingestion permission.
+	Token: aws.String("my-api-token"),
+})
+
+func handler(request events.APIGatewayProxyRequest) (events.APIGatewayProxyResponse, error) {
+	resp, err := http.Get(DefaultHTTPGetAddress)
```

Comment:
can add few lines as example of how customer can send different custom metrics to wavefront.